__tests__/app.test.js

```diff
@@ -121,3 +121,40 @@ describe("handles all bad paths", () => {
     })
   })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: should respond with an array of comments for the given article_id", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(( { body }) => {
+      const { comments } = body
+      expect(comments).toHaveLength(11)
+      expect(comments).toBeSortedBy("created_at", {descending: true})
+      comments.forEach((comment) => {
+        expect(comment.article_id).toBe(1)
+        expect(comment).toHaveProperty("author", expect.any(String))
+        expect(comment).toHaveProperty("body", expect.any(String))
+        expect(comment).toHaveProperty("article_id", expect.any(Number))
+        expect(comment).toHaveProperty("created_at", expect.any(String))
+        expect(comment).toHaveProperty("votes", expect.any(Number))
+      })
+    })
+  })
```

Comment:
There is one other happy path test - what if the article exists but it has no comments? We should return the empty array as these are the comments for that article. You will need to find a way to stop the 404 error being sent in this circumstance.